jasmine.json

```diff
@@ -6,6 +6,5 @@
   "spec_files": [
     "*Test.js"
   ],
-  "random": true,
-  "timeout": 20000
```

Comment:
Why remove timeout? What default timeout does apply?

Reply:
helper.js
```
jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000;
```
It was currently set in to different places. Now it's easier to find and change.